src/libraries/Common/tests/System/Net/Http/Http3LoopbackStream.cs

```diff
@@ -285,9 +282,7 @@ public async Task SendResponseBodyAsync(byte[] content, bool isFinal = true)
 
             if (isFinal)
             {
-                _stream.Shutdown();
-                await _stream.ShutdownCompleted().ConfigureAwait(false);
-                Dispose();
+                _stream.CompleteWrites();
```

Comment:
We no longer do the `ShutdownCompleted`  because we expect that in Dispose? Is there reason to delay that if we know this is final?